net/ptunnel-ng/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ptunnel-ng
+PKG_VERSION:=1.3
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/lnslbrty/ptunnel-ng.git
+PKG_SOURCE_VERSION:=a2eee970b908ab3f898b20a546c6bc0517bd2d17
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
```

Comment:
This repo is properly tagged, please use codeload or at least change it to tar.xz